daffodil-cli/src/main/scala/org/apache/daffodil/Main.scala

```diff
@@ -609,17 +609,8 @@ object Main {
   def retrieveTunables(tunables: Map[String, String], configFileNode: Option[Node]) = {
     val configFileTunables: Map[String, String] = configFileNode match {
       case None => Map.empty
-      case Some(configNode) => {
-        val tunablesOpt = (configNode \ "tunables").headOption
-        tunablesOpt match {
-          case None => Map.empty
-          case Some(tunableNode) => {
-            tunableNode.child.map { n => (n.label, n.text) }.toMap
-          }
-        }
-      }
+      case Some(configNode) => DaffodilTunables.tunablesMap(configNode)
```

Comment:
I think the TDMLRunner also does some trim stuff with tunables to solve this same problem. Probably want to modify that to use this new code so there's one place that deal with parsing tunables from Scala Noes.

Also, I wonder if it would be helpful to add another overloaded `tunablesMap` function that accepts a `File` as a parameter. Seems like that might reduce duplication that both Main and TDMLRunner need to support, so that neither needs to load the config file into XML first?

I don't know if this is necessary, but I wonder if eventually we want a `DaffodilConfig` class that handles parsing and extracting inforation from config files? Again with the goal of reducing duplication between Main and the TDMLRunner. That way the variables and tunables (and maybe future config stuff) that is stored in a config file can all be handled consistently.

Reply:
I think I've addressed this now.